File: trail1/1까지 나누기/divide-by-1.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        int N = int.Parse(Console.ReadLine());
        int i = 1, cnt = 0;
        while(N>1) {
            N /= i;
            cnt++;
            i++;
        }
        Console.WriteLine(cnt);
    }
}

File: trail1/1까지 나누기/divide-by-1.c
#include <stdio.h>

int main() {
    int N;
    scanf("%d", &N);
    int i = 1, cnt = 0;
    while(N > 1) {
        N /= i;
        cnt++;
        i++;
    }
    printf("%d", cnt);
    return 0;
}